modules/_labs/combobox/react/stories/stories.tsx

```diff
@@ -40,6 +48,7 @@ class Autocomplete extends React.Component<
           .splice(0, 5)}
         onChange={this.autocompleteCallback}
         showClearButton={this.props.showClearButton == null ? true : this.props.showClearButton}
+        useLegacyAriaAttributes={boolean('useLegacyAriaAttributes', true)}
```

Comment:
If we want to support both ARIA 1.0 and 1.1 specs, I think we should have 2 different stories that state the difference vs one story with a knob.